samples/peripheral/radio_test/CMakeLists.txt

```diff
@@ -9,6 +9,11 @@ cmake_minimum_required(VERSION 3.13.1)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(NONE)
 
+# Application sources
+target_sources_ifdef(CONFIG_NRF21540_FEM app
+	PRIVATE ../../bluetooth/direct_test_mode/src/fem/nrf21540.c)
+zephyr_include_directories(../../bluetooth/direct_test_mode/src/fem/)
```

Comment:
I think we should make it general-purpose module to avoid such a configuration

Reply:
Maybe we could extract common code between the two test samples to a radio_test submodule, assuming we cannot unify the two samples.

Also I think these should be considered applications rather than samples at this point, and should be moved to the applications folder instead.